packages/core/strapi/lib/core/registries/sanitizers.js

```diff
@@ -0,0 +1,26 @@
+'use strict';
+
+const _ = require('lodash');
+
+const sanitizersRegistry = () => {
+  const sanitizers = {
+    'content-api': {
+      input: [],
+      output: [],
+    },
+  };
+
+  return {
+    get(path) {
+      return _.get(sanitizers, path);
+    },
+    add(path, sanitizer) {
+      this.get(path).push(sanitizer);
+    },
+    has(path) {
+      return _.has(sanitizers, path);
+    },
+  };
```

Comment:
What do you think of adding a way to register new sanitizers? Something/A method to add a new key somewhere in a the sanitizers object?
IMO, it could be useful for plugin creators, etc...